pkg/resources/kafka/kafka.go

```diff
@@ -884,6 +884,12 @@ func (r *Reconciler) reconcileKafkaPvc(log logr.Logger, brokersDesiredPvcs map[s
 				if mountPath == pvc.Annotations["mountPath"] {
 					currentPvc = pvc.DeepCopy()
 					alreadyCreated = true
+					// Checking pvc state, if bounded, so the broker has already restarted and the CC GracefulDiskRebalance has not happened yet,
+					// then we make it happening with status update.
+					if _, ok := r.KafkaCluster.Status.BrokersState[brokerId].GracefulActionState.VolumeStates[mountPath]; !ok &&
+						currentPvc.Status.Phase == corev1.ClaimBound {
+						brokerVolumesState[currentPvc.Annotations["mountPath"]] = v1beta1.VolumeState{CruiseControlVolumeState: v1beta1.GracefulDiskRebalanceRequired}
```

Comment:
Is the value of `currentPvc.Annotations["mountPath"]` the same as `mountPath`? if so could we use here `brokerVolumesState[mountPath] ` ?

Reply:
I have changed it to mountPath. Thank you!